code/src/components/Game.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { Start } from "./Start";
+import { Labyrinth } from "./Labyrinth";
+import { LoadingIndicator } from "./LoadingIndicator";
+
+export const Game = () => {
+  const currentPosition = useSelector((store) => store.game.currentPosition);
+
+  return (
```

Comment:
An idea if you want the loader to cover the screen (not sure if you struggled with that as we did) is to set it up like this:
`return (
    <>
      {loading && <LoadingIndicator />}
      {currentPosition ? <GameBoard /> : <Start />}
    </>
  )`